ironfish/src/wallet/walletdb/assetsValue.ts

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import type { IDatabaseEncoding } from '../../storage/database/types'
+import {
+  ASSET_ID_LENGTH,
+  ASSET_METADATA_LENGTH,
+  ASSET_NAME_LENGTH,
+  ASSET_OWNER_LENGTH,
+  PUBLIC_ADDRESS_LENGTH,
+} from '@ironfish/rust-nodejs'
+import bufio from 'bufio'
+import { BigIntUtils } from '../../utils'
+
+export interface AssetsValue {
```

Comment:
Singular since it represents 1 asset, like the other values.

Reply:
Alright, there's some inconsistency in the blockchain as well. I'll fix that in a separate PR.